crates/oxc_transformer/src/es2015/arrow_functions.rs

```diff
@@ -151,7 +151,7 @@ pub struct ArrowFunctions<'a> {
 impl<'a> ArrowFunctions<'a> {
     pub fn new(options: ArrowFunctionsOptions) -> Self {
         // `SparseStack` is created with 1 empty entry, for `Program`
-        Self { _options: options, this_var_stack: SparseStack::new() }
+        Self { _options: options, this_var_stack: SparseStack::with_capacity(16, 0) }
```

Comment:
Looks like we shouldn't do this in each plugin. Is it possible we use the `new` method and pre-allocate in it?

Reply:
Yes, you're right. I've changed the default to 16 in `StackCapacity`, so it's applied everywhere. I don't understand why `std::vec::Vec` uses 8 as default.